deployments/daemon-set/nginx-ingress.yaml

```diff
@@ -11,6 +11,7 @@ spec:
     metadata:
       labels:
         app: nginx-ingress
+        app.kubernetes.io/core-name: nginx-ingress
```

Comment:
I've never seen `core-name`, can't we use `name`?

Reply:
We're already using `name` in the helm chart, but it's not static because we prepend the release name to it. Thinking about it though, that's not correct - the point is to identify an application, not an instance of it. I'll change that and use name instead.